src/server/db/queries/patterns.ts

```diff
@@ -52,11 +52,15 @@ const destroy = (id: string) =>
 	Query("DELETE FROM patterns WHERE id = ?", [id]);
 
 //PATCH a pattern
-const update = (patternDTO: IPatternTable, id: string, author_id: string) =>
-	Query("UPDATE patterns SET ? WHERE id = ? AND author_id = ?", [
-		patternDTO,
-		id,
-		author_id,
+const update = (patternDTO: {
+	author_id: string;
+	content: string;
+	id: string;
+}) =>
+	Query("UPDATE patterns SET content = ? WHERE id = ? AND author_id = ?", [
```

Comment:
This is probably something that you can do a refactor on later, but strings like this should be kept in consts somewhere else so that they can be changed from a central location more easily

Reply:
This IS the central location